google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/schema/SpannerForeignKeyExporter.java

```diff
@@ -39,14 +40,15 @@ public void init(SpannerDatabaseInfo spannerDatabaseInfo) {
   }
 
   @Override
-  public String[] getSqlDropStrings(ForeignKey foreignKey, Metadata metadata) {
+  public String[] getSqlDropStrings(ForeignKey foreignKey, Metadata metadata,
+      SqlStringGenerationContext context) {
```

Comment:
Memo: a backward incompatible change. Next version should have a major version bump (unless this class is not meant for users).